src/hgvs/pretty/console/chrom_seq_renderer.py

```diff
@@ -0,0 +1,43 @@
+from hgvs.pretty.models import VariantData
+from hgvs.pretty.console.renderer import BasicRenderer
+
+
+class ChromSeqRendered(BasicRenderer):
+
+    def legend(self) -> str:
+
+        if self.orientation < 0 and self.config.reverse_display:
+            arrow = '<-'
+        else:
+            arrow = '->'
+
+        return f"seq    {arrow} : "
+
+    def display(self, data: VariantData) -> str:
+        """colors the ref sequences with adenine (A, green), thymine (T, red), cytosine (C, yellow), and guanine (G, blue)"""
+        from hgvs.pretty.console.constants import ENDC, TBLUE, TGREEN, TRED, TYELLOW
+
+        var_seq = ""
+        for p in data.position_details:
+            c = p.ref
+
+            if not c:
+                var_seq += "."
+                continue
+
+            if self.config.useColor:
```

Comment:
Please consider putting a colormap in BasicRenderer to ensure color consistency in both directions.

Or maybe in constants.py is even better.

Reply:
done